wordpress.yaml

```diff
@@ -3,241 +3,68 @@ description: |
-  - name: update_category
-    description: |-
-      Update an existing category in WordPress site. Only provided fields will be updated.
-      
-      At least one field must be provided to update.
-    params:
-      category_id: The ID of the category to update
-      description: 'New description of the category (accepts HTML tags) - default: None'
-      name: 'New name of the category - default: None'
-      parent_id: 'New parent ID of the category - default: None'
-      slug: 'New slug for the category - default: None'
-  - name: update_media
-    description: |-
-      Update the metadata of a media file in WordPress site.
-      
-      At least one field must be provided to update.
-    params:
-      author_id: 'New author ID for the media file - default: None'
-      media_id: The ID of the media file
-      slug: 'New slug for the media file - default: None'
-      title: 'New title for the media file - default: None'
-  - name: update_post
-    description: |-
-      Update a post in WordPress site. Only provided fields will be updated.
-      
-      Use HTML tags for content formatting. Date must be ISO 8601 format.
-    params:
-      author_id: 'New author ID - default: None'
-      categories: 'New comma-separated list of category IDs - default: None'
-      comment_status: 'New comment status - default: None'
-      content: 'New content for the post (use HTML tags for formatting) - default: None'
-      date: 'New publication date (ISO 8601 format) - default: None'
-      excerpt: 'New excerpt - default: None'
-      featured_media: 'New featured media ID - default: None'
-      format: 'New post format - default: None'
-      password: 'New password for the post - default: None'
-      ping_status: 'New ping status - default: None'
-      post_id: ID of the post to update
-      slug: 'New URL slug - default: None'
-      status: 'New status - default: None'
-      sticky: 'Whether the post should be sticky - default: None'
-      tags: 'New comma-separated list of tag IDs - default: None'
-      title: 'New title for the post - default: None'
-  - name: update_tag
-    description: |-
-      Update an existing tag in WordPress site. Only provided fields will be updated.
-      
-      At least one field must be provided to update.
-    params:
-      description: 'New description of the tag (accepts HTML tags) - default: None'
-      name: 'New name of the tag - default: None'
-      slug: 'New slug for the tag - default: None'
-      tag_id: The ID of the tag to update
-  - name: validate_credential
-    description: |-
-      Validate WordPress credentials by attempting to get current user profile.
-      This is useful for testing authentication before performing other operations.
+repoURL: https://github.com/Automattic/wordpress-mcp
 
 env:
   - name: WordPress Site URL
     description: 'The full URL to your WordPress site (wordpress.com not supported). Must start with http:// or https://.'
```

Comment:
Are we sure that wordpress.com is still not supported for this?

Reply:
Good question. The prerequisite for this MCP server is that the plugin must be installed, which is only possible for WordPress.com users on the Business or Commerce plans. Users on lower-tier plans cannot install plugins on their sites. So even if it’s technically supported, not all WordPress.com users would be able to make it work.